bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox/LayerBox.jsx

```diff
@@ -111,7 +147,10 @@ export const LayerBox = ({ layer, index, locale, mutator }) => {
                                         layer={layer}
                                         locale={locale}
                                         mutator={mutator}
-                                        onChange={styleName => mutator.changeLayerStyle(layer, styleName)}/>
+                                        onChange={styleName => mutator.changeLayerStyle(layer, styleName)}
+                                    />
+                                    </>
+                                    }
                                     <ColAutoRight>
                                         <SelectedLayerDropdown tools={tools} />
                                     </ColAutoRight>
```

Comment:
Should we move all this "GrayRow" stuff to a Footer-component to make this more readable?

Reply:
Moved to LayerBoxFooter component